### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by pbrisbin** [LeifW/xmonad-utils#5]
```
Preprocessing executable 'hhp' for xmonad-utils-0.1.3.2.1...
[1 of 2] Compiling Utils            ( src/Utils.hs, dist/build/hhp/hhp-tmp/Utils.o )
[2 of 2] Compiling Main             ( src/Hhp.hs, dist/build/hhp/hhp-tmp/Main.o )

src/Hhp.hs:57:3: Not in scope: ‘block’

src/Hhp.hs:71:16: Not in scope: ‘unblock’
Failed to install xmonad-utils-0.1.3.2.1
cabal: Error: some packages failed to install:
xmonad-utils-0.1.3.2.1 failed during the building phase. The exception was:
ExitFailure 1
```

GHC 7.8.2, Cabal 1.20.0.1, Arch Linux
